src/views/homepage/components/product-nav/product-nav.module.css

```diff
@@ -0,0 +1,132 @@
+.productNav {
+  --container-max-width: 1240px;
+  --padding-x: 24px;
+  --notice-background-color: var(--token-color-palette-neutral-700);
+
+  position: relative;
+}
+
+.notice {
+  position: relative;
+  margin: 0;
+  padding: 0 var(--padding-x) 12px;
+  background-color: var(--notice-background-color);
+  text-align: center;
+}
+
+.noticeText {
+  margin: 0;
+  padding: 0;
+  color: var(--token-color-palette-neutral-400);
+}
+
+.nav {
+  position: relative;
+  padding: 0 var(--padding-x);
+  background-image: linear-gradient(
+    to bottom,
+    var(--notice-background-color) 24px,
+    var(--token-color-palette-neutral-0) 24px
+  );
+}
+
+.list {
+  list-style: none;
+  margin: 0 auto;
+  padding: 0;
+  max-width: var(--container-max-width);
+  display: grid;
+  grid-template-columns: repeat(2, 1fr);
+  gap: 24px;
+  border-radius: 14px;
```

Comment:
Got it, not seeing that on my end but I went ahead and added it. I put it within the desktop media query since we don't want backgrounds once within the grid.